datasets/dataset_setup.py

```diff
@@ -457,17 +455,14 @@ def download_librispeech(dataset_dir, tmp_dir, train_tokenizer):
       'train-other-500.tar.gz',
   ]
   for tar_filename in tars:
-    wget_cmd = (f'wget --directory-prefix={tmp_librispeech_dir} '
-                f'http://www.openslr.org/resources/12/{tar_filename}')
-    subprocess.Popen(wget_cmd, shell=True)
-    tar_path = os.path.join(tmp_librispeech_dir, tar_filename)
-    subprocess.Popen(f'tar xzvf {tar_path}', shell=True)
```

Comment:
just confirming, we don't need to untar the files?

Reply:
The untar now happens together with the wget.